components/Input/Input.tsx

```diff
@@ -178,72 +131,143 @@ const StyledInput = styled('input', {
   },
   defaultVariants: {
     size: 'medium',
-    cursor: 'default'
+    cursor: 'default',
   },
   compoundVariants: [
     {
       startAdornment: true,
       size: 'small',
       css: {
         paddingInlineStart: 'calc($2 + 15px)',
-      }
+      },
     },
     {
       startAdornment: true,
       size: 'medium',
       css: {
         paddingInlineStart: 'calc($3 + 15px)',
-      }
+      },
     },
     {
       startAdornment: true,
       size: 'large',
       css: {
         paddingInlineStart: 'calc($3 + 15px)',
-      }
+      },
     },
     {
       endAdornment: true,
       size: 'small',
       css: {
         paddingInlineEnd: 'calc($2 + 15px)',
-      }
+      },
     },
     {
       endAdornment: true,
       size: 'medium',
       css: {
         paddingInlineEnd: 'calc($3 + 15px)',
-      }
+      },
     },
     {
       endAdornment: true,
       size: 'large',
       css: {
         paddingInlineEnd: 'calc($3 + 15px)',
-      }
+      },
     },
-  ]
+  ],
 });
 
 const InputWrapper = styled('div', {
+  // Reset
+  outline: 'none',
+  lineHeight: 0,
+
   position: 'relative',
-  backgroundColor: '$wrapperBg',
+  backgroundColor: '$inputBg',
+  // cursor: 'auto',
+
+  '&::before': {
+    boxSizing: 'border-box',
+    content: '""',
+    position: 'absolute',
+    top: 0,
+    right: 0,
+    bottom: 0,
+    left: 0,
+  },
+  '&::after': {
+    boxSizing: 'border-box',
+    content: '""',
+    position: 'absolute',
+    top: 0,
+    right: 0,
+    bottom: 0,
+    left: 0,
+  },
+
+  '&:focus-visible': {
+    '&::before': {
+      backgroundColor: '$inputFocusBg',
+    },
+    '&::after': {
+      backgroundColor: '$primary',
+      opacity: 0.15,
+    },
+  },
+
+  '@hover': {
+    '&:hover': {
```

Comment:
You could simply add `:not(:disabled)` to avoid resetting it in `disabled` variant.